tracecat/webhooks/router.py

```diff
@@ -59,10 +63,12 @@ async def incoming_webhook(
         trigger_type=TriggerType.WEBHOOK,
     )
     if echo:
+        if empty_echo:
+            return Response(status_code=200)
         try:
             response["payload"] = await request.json()
         except Exception as e:
-            logger.warning("Failed to echo request payload body", error=str(e))
+            logger.warning("Failed to decode request payload body", error=str(e))
```

Comment:
This code path is only related to echo and is an internal facing log
```suggestion
            logger.warning("Failed to echo request payload body", error=str(e))
```